src/test/cypress/e2e/exam/ExamAssessment.cy.ts

```diff
@@ -10,10 +10,17 @@ import textSubmission from '../../fixtures/text_exercise_submission/text_exercis
 import multipleChoiceQuizTemplate from '../../fixtures/quiz_exercise_fixtures/multipleChoiceQuiz_template.json';
 import { makeSubmissionAndVerifyResults } from '../../support/pageobjects/exercises/programming/OnlineEditorPage';
 
-// requests
-const courseManagementRequests = artemis.requests.courseManagement;
+// Users
+const users = artemis.users;
+const admin = users.getAdmin();
+const instructor = users.getInstructor();
+const studentOne = users.getStudentOne();
+const tutor = users.getTutor();
```

Comment:
```suggestion
const admin = users.getAdmin();
const instructor = users.getInstructor();
const tutor = users.getTutor();
const studentOne = users.getStudentOne();
```

All user consts before were sorted from highest to lowest role

Reply:
Done, thanks for the hint!